engine/gamesys/src/gamesys/scripts/script_resource.cpp

```diff
@@ -1003,6 +1016,131 @@ static int SetTexture(lua_State* L)
     return 0;
 }
 
+/*# get texture info
+ * Gets texture info from a texture resource path or a texture handle
+ *
+ * @name resource.get_texture_info
+ *
+ * @param path [type:hash|string|handle] The path to the resource or a texture handle
```

Comment:
Note to reviewer: the function supports both a path and a handle because I think it will make sense to query handles created from elsewhere, but now that I think about it maybe it should be in a different namespace?